src/mono/wasm/runtime/library_mono.js

```diff
@@ -1817,6 +1823,9 @@ var MonoSupportLib = {
 
 		// Used by the debugger to enumerate loaded dlls and pdbs
 		mono_wasm_get_loaded_files: function() {
+			if (!this.mono_wasm_set_is_debugger_attached)
+				this.mono_wasm_set_is_debugger_attached = Module.cwrap ('mono_wasm_set_is_debugger_attached', 'void', ['bool']);
+			this.mono_wasm_set_is_debugger_attached (true);
```

Comment:
There is no easy way to check the attached state from js now though right?

Reply:
I searched if there is an easy way to get debugger state from js and I didn't find.
And another thing, if the debugger is attached but it's not using our ProxyDriver we don't want to send any information right?

Reply:
correct